cmd/gokr-packer/write.go

```diff
@@ -178,6 +180,70 @@ var (
 	}
 )
 
+type kernelManifest struct {
+	KernelGlobs     []string `toml:"kernel_globs"`
```

Comment:
It’s probably a good idea to use a separate kernel package for odroid altogether, even if it’s derived from github.com/gokrazy/kernel. We can then just expand the globs and don’t need to make the globs configurable.

Reply:
Right, I'm using https://github.com/anupcshan/gokrazy-odroidxu4-kernel for the Odroid kernel. But yeah, it's simpler to just add `boot.scr` to kernelGlobs than adding a field in the TOML file. Done that.